doc/debug-messages.txt

```diff
@@ -68,3 +68,4 @@
 59 Shutdown: Digest authentication.
 60 Shutdown: Negotiate authentication.
 61 Shutdown: NTLM authentication.
+62 Error negotiating SSL connection on FD ...
```

Comment:
To avoid changing this ID in customer configurations, I suggest using the ID if this message in future master code.

AFAICT, you can even avoid creating additional gaps in bag14 numbering if you use ID 62 in bag14/master _and_ increase the current master PR numbering for the errors quoted below by one.

```
62 Resuming indexing cache_dir # ... from ...
63 Indexing cache entries: ...
```

Reply:
I used the 62 number for the correspondent message in master.